Mastodon/Protocol/Provider/DataSourceProvider+StatusTableViewCellDelegate.swift

```diff
@@ -618,9 +618,7 @@ extension StatusTableViewCellDelegate where Self: DataSourceProvider & AuthConte
                     provider: self,
                     account: account
                 )
-            case .notification:
-                assertionFailure("TODO")
-            case .hashtag(_):
+            case .notification, .hashtag(_), .notificationBanner(_):
                 assertionFailure("TODO")
```

Comment:
Basically same question as here: https://github.com/mastodon/mastodon-ios/pull/1319/files#diff-2bf0f1a0f7802c1cb4943a5515edda158dd617c42be9e2c81465709a1e37499cR518

If this is never invoked maybe a `break` would be sufficient?